LayoutTests/imported/w3c/web-platform-tests/css/css-view-transitions/rotated-cat-off-top-edge.html

```diff
@@ -4,7 +4,7 @@
 <link rel="help" href="https://drafts.csswg.org/css-view-transitions-1/">
 <link rel="author" href="mailto:vmpstr@chromium.org">
 <link rel="match" href="rotated-cat-off-top-edge-ref.html">
-<meta name="fuzzy" content="rotated-cat-off-top-edge-expected.html:0-5;0-1500">
+<meta name="fuzzy" content="maxDifference=0-140; totalPixels=0-22000">
```

Comment:
This is quite a large difference! Do they look visually identical?

I think we should probably dig into this a bit more before upstreaming this one.

Reply:
> This is quite a large difference! Do they look visually identical?

Yes.

iOS has a large maxDifference but a smaller number of totalPixels, macOS has the opposite, not sure I can specify two different fuzzy data for platforms
so I just took the max of both